src/Malawi_cor_labels_GEE script.js

```diff
@@ -0,0 +1,342 @@
+         [33.95703682527283, -12.237925904996903],
+         [33.95257362947205, -12.241700489480108],
+         [33.95102867707947, -12.228866682206075],
+         [33.93823990449646, -12.225259683004753],
+         [33.93206009492615, -12.23566112841648],
+         [33.93497833833435, -12.24983663252191],
+         [33.9498270474408, -12.248997866483007],
+         [33.9142931424115, -12.228363382946103],
+         [33.911718221757205, -12.242623157481672],
+         [33.98742088899353, -12.254869172764044],
+         [33.953431936356814, -12.262921037431857],
+         [33.95806679353455, -12.220142692759165]]),
+    Sentinel2 = ee.Image("projects/ee-aasareansah/assets/Sent_Sept_Comp_Malawi"),
+    Malawi_EPA = ee.FeatureCollection("projects/ee-aasareansah/assets/NASA_Harvest/Malawi_EPAs"),
+    Malawi_TA = ee.FeatureCollection("projects/ee-aasareansah/assets/NASA_Harvest/Malawi_TAs"),
+    Admin = ee.FeatureCollection("projects/ee-aasareansah/assets/NASA_Harvest/Malawi_admin2");
+
+
+//Rename b1 as True value 
+var upd_name = malawi_binary.select('b1').rename('Wrong value');
+
+var leftMap = ui.Map();
+var layer1 = ui.Map.GeometryLayer([], 'Geometry Layer', 'red');
+var drawingTools1 = leftMap.drawingTools();
+drawingTools1.layers().add(layer1);
+
+var rightMap = ui.Map();
+var layer2 = ui.Map.Layer(ee.FeatureCollection([]), {'color': 'red'}, 'Points');
+
+// Link two layers together
+drawingTools1.onDraw(function(geometry, layer) {
+  if (layer === layer1) {
+    layer2.setEeObject(layer.getEeObject());
+  }
+});
+drawingTools1.onEdit(ui.util.debounce(function(geometry, layer) {
+  if (layer === layer1) {
+    layer2.setEeObject(layer.getEeObject());
+  }
+}, 100));
+drawingTools1.onErase(function(geometry, layer) {
+  if (layer === layer1) {
+    layer2.setEeObject(layer.getEeObject());
+  }
+});
+
+//Set map center using this long lat and zoom level
+rightMap.setCenter(33.78, -12.27, 11);
+
+// Make the default basemap the satellite layer
+rightMap.setOptions('satellite');
+
+//Specify the point as the only drawing tool 
+drawingTools1.setDrawModes(['point']);
+
+//Assign specific palettes to the cropland map
+var palettes = require('users/gena/packages:palettes');
+var palette = palettes.cmocean.Speed[7];
+
+// Use this empty image for paint().
+var empty = ee.Image().byte();
+
+var Admin_color = empty.paint({
+featureCollection: Admin,
+color: 1,
+width: 2
+});
+
+var EPA_color = empty.paint({
+featureCollection: Malawi_EPA,
+color: 1,
+width: 2
+});
+
+var TA_color = empty.paint({
+featureCollection: Malawi_TA,
+color: 1,
+width: 2
+});
+
+//Display the cropland map on the left panel
+var vis_crop = {min: 0, max: 1.0, palette: palette.slice(0,-2)};
+leftMap.addLayer(malawi_binary, vis_crop, 'Malawi cropland');
+leftMap.addLayer(Admin_color, {palette: 'CE33FF'}, 'Malawi_Admin');
+leftMap.addLayer(EPA_color, {palette: '0096FF'}, 'Malawi_EPA', false);
+leftMap.addLayer(TA_color, {palette: 'black'}, 'Malawi_TA', false);
```

Comment:
It's not immediatley obvious to me what EPA and TA means here. Perhaps writing out the word would be better.
Also "Malawi" should just be a string variable declared somewhere above and used here so it can be changed easily for other countries / regions.

Reply:
@ivanzvonkov, Can you share the shapefile that was used to generate the Malawi cropland maps? I explored the ne_10m_admin_0_countries shapefile so that one can easily change the country name in the code but this layer didn't align with the cropland extent. I also checked the Malawi boundary generated when creating the CEO project but I had the same issue. Adding a screenshot...

![image](https://user-images.githubusercontent.com/110104000/204614150-35512f03-0c87-4d11-865f-b57c939f7e1f.png)

Reply:
haha looks like I made almost the same comment today